README.md

```diff
@@ -438,6 +438,8 @@ We use [MDX](https://mdxjs.com), which is like normal markdown but adds extensio
 
 Refer to the [Common Markdown syntax](https://commonmark.org/) for a primer on Markdown. The below guide focuses on the other features you can use when writing docs.
 
+If you need to deploy a file from the `/tutorials/` directory to the Learning Platform, see the instructions in the [tutorials directory README](https://github.com/Qiskit/documentation/blob/main/tutorials/README.md).
```

Comment:
What about instead having a new section `## How to deploy docs` before `# How to write the documentation`? For normal docs, it links to https://github.ibm.com/IBM-Q-Software/iqp-channel-docs#syncing-content-with-open-source-repo. For tutorials, it has what you wrote

Reply:
Sounds good!